mino/minogrpc/example_test.go

```diff
@@ -57,14 +57,14 @@ func ExampleRPC_Call() {
 }
 
 func ExampleRPC_Stream() {
-	mA, err := NewMinogrpc(ParseAddress("127.0.0.1", 0), tree.NewRouter(NewAddressFactory()))
+	mA, err := NewMinogrpc(ParseAddress("127.0.0.1", 20000), tree.NewRouter(NewAddressFactory()))
```

Comment:
Is there a particular reason for changing that ? Using port 0 makes the system assign a random free port.

Reply:
I think the reason this was added was to make it easier to debug this Example. It takes a few more breakpoints to figure out the port for `mA` and `mB` but this way it's consistently `20000` and `30000`.